src/contribute/work-in-progress.md

```diff
@@ -25,7 +25,7 @@ Current branches include:
 | Feature | Branch | JIRA Component | More Info |
 | ---- | ---- | ---- | ---- |
 | Apache Gearpump Runner | [gearpump-runner](https://github.com/apache/beam/tree/gearpump-runner) | [runner-gearpump](https://issues.apache.org/jira/browse/BEAM/component/12330829) | [README](https://github.com/apache/beam/blob/gearpump-runner/runners/gearpump/README.md) |
-| Python SDK | [python-sdk](https://github.com/apache/beam/tree/python-sdk) | [sdk-py](https://issues.apache.org/jira/browse/BEAM/component/12328910) | [README](https://github.com/apache/beam/blob/python-sdk/sdks/python/README.md) |
+| Python SDK | [python-sdk](https://github.com/apache/beam/tree/master/sdks/python) | [sdk-py](https://issues.apache.org/jira/browse/BEAM/component/12328910) | [README](https://github.com/apache/beam/blob/master/sdks/python/README.md) |
```

Comment:
This is list of feature branches -- Python SDK like should be *removed* from here, not updated.

Regarding the README.md file -- they are not discoverable if they are deep in the structure, so they make little impact and it is hard to keep them up-to-date. Most important information should go to the top-level README, and everything else to the website under Documentation -> SDKs -> Python SDK. This is nicely formatted, searchable, visible, etc.